src/server.ts

```diff
@@ -380,12 +380,17 @@ const createParticipant = async (tag: string): Promise<ParticipantInfo> => {
 const deleteSession = async () => {
   if (sessionId) {
     try {
-      await webRTCController.deleteSession(accountId, sessionId);
       console.log(`Deleted WebRTC session: ${sessionId} `);
       sessionId = "";
     } catch (e) {
+      let error: any = e;
       console.log("failed to delete session", sessionId);
-      console.log("error", e.response.status, e.response.data, e.config.url);
+      console.log(
```

Comment:
Does this look okay as displayed in the console?

Reply:
not sure what 'OK ' looks like, but it is clear and comprehensive.  it exposes perhaps too much data.

Reply:
Mainly wanting to see if it works and displays what you want it to